messages/Config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
 	"GeneratingConfigFromPilot": "Generating config file by copying pilot config file. If this is undesirable, delete or rename `%s`.",
-	"InvalidStringArrayValue": "Invalid value for property '%s' in Config for %s. Expected an array of strings: %s",
-	"InvalidBooleanValue": "Invalid value for property '%s' in Config for %s. Expected a boolean: %s",
-	"InvalidNumberValue": "Invalid value for property '%s' in Config for %s. Expected a number: %s",
+	"InvalidStringArrayValue": "Specify a valid property for '%s' in %s for %s. Supply an array of strings: %s.",
+	"InvalidBooleanValue": "Specify a valid property for '%s' in %s for %s. Supply a boolean: %s.",
+	"InvalidNumberValue": "Specify a valid property for '%s' in %s for %s. Supply a number: %s.",
```

Comment:
Same here: config is missing. See Google doc. Should read: "Specify a valid property for ‘%s’ in config for %s. Supply a number: %s."

Reply:
@rmohan20 So just double-checking this is following the same revised syntax as above? We need to update the UI text G doc then.

Reply:
@teresa-allen-sfdc Yes, this would need the same update.